website/docs/r/lb_target_group.html.markdown

```diff
@@ -85,6 +85,7 @@ The following arguments are supported:
 * `slow_start` - (Optional) Amount time for targets to warm up before the load balancer sends them a full share of requests. The range is 30-900 seconds or 0 to disable. The default value is 0 seconds.
 * `stickiness` - (Optional, Maximum of 1) Stickiness configuration block. Detailed below.
 * `tags` - (Optional) Map of tags to assign to the resource. If configured with a provider [`default_tags` configuration block](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.
+* `target_failover` - (Optional) Target failover block. Detailed below. Only applicable for Gateway Load Balancer target groups.
```

Comment:
It's helpful for readers of the documentation to have a link to the block documentation below. We also prefer that "See below" sections come as the last sentence

```suggestion
* `target_failover` - (Optional) Target failover block. Only applicable for Gateway Load Balancer target groups. See [target_failover](#target_failover) for more information. 
```